platforms/android/tangram/src/main/java/com/mapzen/tangram/MapController.java

```diff
@@ -1270,6 +1299,24 @@ public void onResponse(Call call, Response response) throws IOException {
         return true;
     }
 
+    // Called from JNI on worker or render-thread.
+    void sceneReadyCallback(final int sceneId, final boolean success, final SceneUpdateError error) {
```

Comment:
Should be setting a `SceneID` for a MapController and check for it to equal to the one set in the callback? To make sure the callback being called is for the apt sceneLoad/sceneUpdate?